pkg/asset/ignition/machine/node.go

```diff
@@ -11,7 +11,8 @@ import (
 	baremetaltypes "github.com/openshift/installer/pkg/types/baremetal"
 	openstacktypes "github.com/openshift/installer/pkg/types/openstack"
 	openstackdefaults "github.com/openshift/installer/pkg/types/openstack/defaults"
-	"github.com/openshift/installer/pkg/types/ovirt"
+	ovirttypes "github.com/openshift/installer/pkg/types/ovirt"
```

Comment:
Intentionally included here?

Reply:
@rgolangh FYI if so.

Reply:
I was standardizing the imports to use `*types`.  I can back that out if it is an issue.

Reply:
lets leave it, just wanted to make sure that it wasn't an oversight